src/bench/base58.cpp

```diff
@@ -22,7 +22,11 @@ static void Base58Encode(benchmark::State& state)
         }
     };
     while (state.KeepRunning()) {
+#ifndef _MSC_VER
         EncodeBase58(buff.begin(), buff.end());
+#else
+		EncodeBase58(buff.begin()._Ptr, buff.end()._Ptr);
+#endif
```

Comment:
This would be portable and avoid the ifdef:
```
EncodeBase58(buff.data(), buff.data() + buff.size()
```
(another option would be to make buff a vector, and you could just do `EncodeBase58(buff)`)